pkg/rehearse/jobs.go

```diff
@@ -164,7 +187,7 @@ func inlineCiOpConfig(job *prowconfig.Presubmit, targetRepo string, ciopConfigs
 
 // ConfigureRehearsalJobs filters the jobs that should be rehearsed, then return a list of them re-configured with the
 // ci-operator's configuration inlined.
-func ConfigureRehearsalJobs(toBeRehearsed config.Presubmits, ciopConfigs config.CompoundCiopConfig, prNumber int, loggers Loggers, allowVolumes bool) []*prowconfig.Presubmit {
+func ConfigureRehearsalJobs(toBeRehearsed map[string][]prowconfig.Presubmit, ciopConfigs config.CompoundCiopConfig, prNumber int, loggers Loggers, allowVolumes bool, cmClient coreclientset.ConfigMapInterface, templates map[string]*templateapi.Template) []*prowconfig.Presubmit {
```

Comment:
Please do not change the `config.Presubmits` type back to `map[string][]prowconfig.Presubmit`

Reply:
oh. sry lost in the rebase :)